src/main/java/io/fabric8/maven/docker/StopMojo.java

```diff
@@ -105,4 +112,27 @@ private boolean invokedTogetherWithDockerStart() {
         Boolean startCalled = (Boolean) getPluginContext().get(CONTEXT_KEY_START_CALLED);
         return startCalled != null && startCalled;
     }
+
+    private Set<Network> getUnusedCustomNetworks(QueryService queryService, PomLabel pomLabel) throws DockerAccessException {
+        if (!createCustomNetworks) {
+            return Collections.emptySet();
+        }
+        Set<Network> customNetworks = new HashSet<>();
+        for (ImageConfiguration image : getImages()) {
+            final NetworkingMode networkingMode = image.getRunConfiguration().getNetworkingMode();
+            if (networkingMode.isCustomNetwork()) {
+                Network network = queryService.getNetworkByName(networkingMode.getCustomNetwork());
+                if (network != null) {
+                    customNetworks.add(network);
+                    for (Container container : getContainersToStop(queryService, image)) {
+                        if (!shouldStopContainer(container, pomLabel)) {
+                            // it's sill in use don't collect it
+                            customNetworks.remove(network);
+                        }
+                    }
+                }
+            }
+        }
+        return customNetworks;
+    }
```

Comment:
Maybe one could use the `options` field in `NetworkCreateConfig` to pass in a tag for dmp. This would work if the network driver are supposed to ignore unknown options (but thats not documented and probably need to be tried out).